From eb92ab00bdbdafd4db7fdeec0100ec803990edab Mon Sep 17 00:00:00 2001
From: Andy Duan <fugang.duan@nxp.com>
Date: Wed, 27 Jun 2018 09:56:13 +0800
Subject: [PATCH 4070/5242] MLK-18695 ARM: dts: imx7d-sdb: fix module loadable
 issue

commit  3898a21d257bc537cb8f529099d63e19497dd984 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add property "cap-power-off-card" to fix the module loadable issue.

Reviewed-by: Frank Li <frank.li@nxp.com>
Signed-off-by: Fugang Duan <fugang.duan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-sdb.dts |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index 2a7e562..060db6b 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -735,6 +735,7 @@
 	mmc-pwrseq = <&usdhc2_pwrseq>;
 	fsl,tuning-step = <2>;
 	pm-ignore-notify;
+	cap-power-off-card;
 	status = "okay";
 
 	brcmf: bcrmf@1 {
-- 
1.7.9.5

